### DISCUSSION TITLE
Use of Whisper model in the repo

### DISCUSSION THREAD

**Discussion by ramkumarkoppu** [keras-team/keras-hub#1151] [Q&A]
Hi, I would like to explore the possibilities of OpenAI Whisper model for custom data. There are two paths available. One is hugging face blog on fine tuning PyTorch whisper model with the Colab notebook example. Or I could use whisper model from Keras-nlp repo. But I don't know how to use it. @abheesht17: I knew you recently done work on this model in Keras-NLP by your commits. do you have any example code of keras-NLP whisper to recommend? Is this model in the keras-nlp fine tuneable? Thanks in advance.

**Comment by abheesht17** [comment]
@ramkumarkoppu - Unfortunately, Whisper isn't complete yet. There are a few bugs we have to investigate

**Reply by ak3ra** [reply]
@abheesht17 I see whisper is now [here](https://github.com/keras-team/keras-nlp/tree/master/keras_nlp/models/whisper). But cant find a way to use it on a custom dataset like @ramkumarkoppu is asking. 
would be keen on working on this if not yet there. Any pointers are welcome.

**Comment by innat-asj** [comment]
https://github.com/sanchit-gandhi/whisper-jax
